include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -361,13 +377,30 @@ struct pairwise_alignment_edit_distance_unbanded
         else
             large_patterns();
 
-        return *this;
+        if constexpr(is_global)
+            _best_score = _score;
+    }
+
+public:
+
+    //!\brief \copydoc _compute
+    pairwise_alignment_edit_distance_unbanded run() &&
```

Comment:
if you return the type as `&&` you could save a call to the move constructor in case you use it in a forwarding expression.

Reply:
example?

Reply:
ah you mean the return type should be `pairwise_alignment_edit_distance_unbanded &&`

Reply:
I thought this will automatically be done by return value optimization.